clients/cobol-lsp-vscode-extension/snippets.json

```diff
@@ -1982,5 +1982,108 @@
 			"$0"
 		],
 		"scope": "cobol"
-	}
+	},
+    "IDMS-CONTROL SECTION ": {
+        "prefix": "IDMS-CONTROL-SECTION",
+        "body": [
+            "IDMS-CONTROL SECTION.",
+            "PROTOCOL. MODE IS ${1:operating-mode}."
+        ],
+        "description": "[IDMS] idms-control section",
+        "scope": "cobol"
+    },
+    "SCHEMA SECTION ": {
```

Comment:
Some names end on spaces. Can you delete those tailing spaces?

Reply:
I removed the spaces